docs/wiki/The CI environment.md

```diff
@@ -9,6 +9,7 @@ This section provides an overview of the design and principles the CARML Continu
 - [Pipeline design](./The%20CI%20environment%20-%20Pipeline%20design)
     - [Static validation](./The%20CI%20environment%20-%20Static%20validation)
     - [Deployment validation](./The%20CI%20environment%20-%20Deployment%20validation)
+      - [Deployment removal](./The%20CI%20environment%20-%20Deployment%20removal)
```

Comment:
I'd suggest changing the title to
```suggestion
      - [Deployment history cleanup](./The%20CI%20environment%20-%20Deployment%20history%20cleanup)
```
and renaming the corresponding page. That's to avoid any confusion with the resource removal step